### ISSUE TITLE
LFS fails to upload videos files to GitHub

### ISSUE THREAD

**Issue by dselman** [git-lfs/git-lfs#3233]
```
git --version
git version 2.15.2 (Apple Git-101.1)

git lfs --version
git-lfs/2.5.1 (GitHub; darwin amd64; go devel +bd98a81dc2 Thu Jul 26 17:13:12 2018 +0000; git e1628484)
```

Steps:
1. Purchase a GH data pack (50 GB) for the org: https://github.com/accordproject/tech-wg
1. Create a new empty GH repo
2. Clone it
3. `git lfs install`
4. `git lfs track "*.mov"`
5. `git lfs track "*.mp4"`
6. git commit and push the `.gitattributes` file
7. Add about 800 MB of mp4 and mov files
8. `git commit` the files
9. `git push`
10. See all the files being uploaded. Once upload is complete the push then fails with the error below:

```
Daniels-MacBook-Pro-5:tech-wg dselman$ git push
LFS upload failed:cts:   5% (1/19), 786 MB | 1.3 MB/s                                                                                                                                                               
  (missing) meetings/accord-tech-wg-2018-01-24.mp4 (69a337e3db0af667d675bb098c54a1608c028fdc65166d7b46005f19696fe031)
  (missing) meetings/accord-tech-wg-2018-02-21.mp4 (fb86479939b1b40a1d6026ae37cb622c42378691141e4f4ad2d90b13021c34d8)
Uploading LFS objects:   5% (1/19), 786 MB | 1.3 MB/s, done
  (missing) meetings/accord-tech-wg-2018-08-15.mp4 (cc0a7f977ccc15ff0bd4924e6c8fe3b0ff48aedd5675e792ac014fd032fcde01)
  (missing) meetings/accord-tech-wg-2018-08-22.mp4 (1db0b6ffff2c546337f1d6627f6481fb6e27a2aa380e788ef37b89a0fcd5d8d3)
  (missing) meetings/accord-tech-wg-2018-05-02.mp4 (90dd21eb0459c7764cbfcce0ee9e3be4a4832774eee1b89ed94405e0b39a72f8)
  (missing) meetings/accord-tech-wg-2017-12-06.mp4 (dd8c08112c5842fb2aae25b26bf9b6fa8725622ffa2501c0d61fdfe4812484d2)
  (missing) meetings/accord-tech-wg-2018-01-31.mp4 (51626b0d34781e3a962206f62492a60854949588aaeb0a5a1d90d30ad321566c)
  (missing) meetings/accord-tech-wg-2018-02-28.mp4 (26d30e3f8ebd2991ca2b9a2ca5d12accf7927340a4ed9b03514b41c813b89c29)
  (missing) meetings/accord-tech-wg-2018-03-28.mp4 (736d26ebf31e88b4e946c7944b56a468712a91a92ba86869694a074ee49e0231)
  (missing) meetings/accord-tech-wg-2018-08-29.mp4 (8d3a17af350ecb210c552ae39d4f075ecca8a6abc66a2654ea7446e961e692f2)
  (missing) meetings/accord-tech-wg-2018-01-17.mp4 (52a2d948745007096c583560380ae0e61044e52a71bf255cf3cc7084d7dec5ca)
  (missing) meetings/accord-tech-wg-2017-11-22.mp4 (c713d62e4737d75efb2bcf9899384a632c440395ec710234be3bd360f2344713)
  (missing) meetings/accord-tech-wg-2018-09-05.mp4 (e924789943e54ffa3963597f64f68db43c060054b022c4ba5157cc20f5002e6f)
  (missing) meetings/accord-tech-wg-2018-03-07.mp4 (2becfce18b140eda9d0a475d2c5db822d6182d45e0314b9d79bfef9240a69290)
  (missing) meetings/accord-tech-wg-2018-03-15.mp4 (367ad0457f90d82877c0a3d76b1e40e8b4d543acc8a73e310d100c8633b6e4ac)
  (missing) meetings/accord-tech-wg-2017-11-29.mp4 (3bc3e1e42be1e108a07fcba79bb5f360ff306616366acedc7478366a6f4d01d3)
  (missing) meetings/accord-tech-wg-2018-05-30.mp4 (09dc311e213821db2d9037da76ed253c56a7e345617174883cbc85a803bf9646)
  (missing) meetings/accord-tech-wg-2017-12-13.mp4 (3483e39aae8fe335a1bdefc568ba31650c88c6625a0deffb80601de51823e9d4)
hint: Your push was rejected due to missing or corrupt local objects.
hint: You can disable this check with: 'git config lfs.allowincompletepush true'
error: failed to push some refs to 'https://github.com/accordproject/tech-wg.git'
```

**Comment by ttaylorr** [comment]
Hi, Dan. Thanks for opening this issue, and I'm sorry that you are
experiencing some trouble.

The issue that you saw indicates that objects that you were trying to
push to the server did not exist in your local .git/lfs/objects cache.
This can happen for a variety of reasons, the most prevalent of which is
that you did not run "git lfs install" from within your repository.

I see that you did mention that you had run "git lfs install", but I
want to make sure that you had run it inside of your repository, too.

Beyond that, can you list the contents of .git/lfs/objects to see if the
files are in fact there? If they are, it's likely that this is more
related to a bug in Git LFS not pushing objects that it should, instead
of a configuration problem.

**Comment by ttaylorr** [comment]
I'm going to close this issue because it has been inactive for a little while.

If there are any problems in the future, please don't hesitate to let us know -- we'd be more than happy to help out. Thanks!

**Comment by rogerthat94** [comment]
I just had this problem. It turned out that I was trying to add the placeholders for files, rather than the actual files themselves. Try running ls -l in the directory with the files. If they are all really small, try opening them with cat. In my case they all just contained the following text:
`version https://git-lfs.github.com/spec/v1
oid sha256:<The hash>
size <The size>
`

**Comment by ttaylorr** [comment]
@rogerthat94 were you able to resolve your issue? Please make sure that you have Git LFS installed and configured correctly (those details can be found at git-lfs.github.com). If that doesn't resolve it, please don't hesitate to open a new issue. Thanks!

**Comment by MicSokoli** [comment]
I just had the same experience as rogerthat94. Googled the error, found this page, and I also had placeholders instead of the actual files. Replacing the placeholders with the actual files worked.

**Comment by gwerbin** [comment]
I just got the same error, but in my case the files are present and have the correct size.

This happened after renaming a remote. Is it possible that renaming a remote could introduce some kind of internal inconsistency in the Git LFS cache?

**Comment by bk2204** [comment]
Git LFS will optimize pushes based on the remote tracking branches. Any commits that are on a remote tracking branch for the remote you're pushing to will not be considered as sources of LFS objects to be pushed. If that isn't a valid optimization for your case (e.g., someone hasn't pushed the LFS objects for those commits), then you'll need to run `git lfs push --all` to make sure they're uploaded.

**Comment by gwerbin** [comment]
That makes sense and thanks for the tip about `--all`.

That isn't the situation I had, in this case. I had cloned from a remote that used to be called `origin`, then got renamed to `archive`, and I tried to push to a *new* branch called origin.

**Comment by bk2204** [comment]
@gwerbin That sounds like a slightly different issue than what's being reported here. Could you open a new issue with reproduction steps (and the information requested in the contributing docs)?

**Comment by jvsonjohnson** [comment]
type this in the command line with the file type at the end
git lfs migrate import --include="*.mp4" 

then commit and push again

**Comment by HakimSkaik** [comment]
it keeps saying not in a git repository
